modules/apps/transfer/types/codec.go

```diff
@@ -41,9 +41,6 @@ var (
 	// The actual codec used for serialization should be provided to x/ibc transfer and
 	// defined at the application level.
 	ModuleCdc = codec.NewProtoCodec(codectypes.NewInterfaceRegistry())
-
-	// AminoCdc is a amino codec created to support amino json compatible msgs.
-	AminoCdc = codec.NewAminoCodec(amino)
```

Comment:
ditto on the above registerlegacyaminocodec

Reply:
I did this, but please know that it infolved removing a few other calls and functions, I'll put the commit here. 

Hm, I think that this is a no.....